src/Sequencer4Module.cpp

```diff
@@ -1,26 +1,23 @@
-#include "Squinky.hpp"
 #include "Sequencer4Module.h"
-#include "Sequencer4Widget.h"
 
-#include "SqStream.h"
 #include "MidiSong4.h"
-
+#include "Sequencer4Widget.h"
+#include "SqStream.h"
+#include "Squinky.hpp"
```

Comment:
There appear to be a lot of changes here, but I think that's just because this file have never been autoformatted by VSC before.